2014-15/C06 Death fixes.sps

```diff
@@ -5,12 +5,12 @@
 * Determine the most appropriate death date to use.
 ***************************************************************************************************************************.
 *Get file from previous syntax. 
-Get file = !File + "temp-source-episode-file-5-" + !FY + ".zsav". 
+Get file = !Year_dir + "temp-source-episode-file-5-" + !FY + ".zsav". 
 
 * Only keep relevant variables.
 match files
     /file  = *
-    /table = !Extracts_Alt + "All Deaths.zsav"
+    /table = !Deaths_dir + "All Deaths.zsav"
```

Comment:
Needs updating to the new name of the deaths lookup (just in case you're not already doing it!)

* For all years.